### ISSUE TITLE
Enrichment analysis tool header/footer update

### ISSUE THREAD

**Issue by raymond91125** [WormBase/caltech-curation-services#38]
Currently the enrichment analysis tool CGI
**<wobr1.caltech.edu:/home/azurebrd/git/caltech-tools/tea.cgi>**
fetches the wormbase header/footer from tazendra and it needs to be changed to 
**<https://caltech-curation.textpressolab.com/files/pub/wormbaseheader/WB_header_footer.html>**
when the production server goes live.

**Comment by valearna** [comment]
@azurebrd it looks like the headers from tazendra and the dockerized system are old (the release versions are not automatically updated). Can you take a look?

**Comment by azurebrd** [comment]
@valearna Yeah, this seems to break really often.  And by really often I mean, every couple or few releases.  Eventually someone notices, and I update the script.  It might be easier to remove the version from the local header, like we did for the Alliance version that can't be automated.  Or maybe at some point someone said they could change it manually, I don't recall.

**Comment by azurebrd** [comment]
@valearna well.  We could ask people what they want.  The current script is trying to figure out what the WB header and footer and javascript are and scrape it and generate a file that can be added to every form.  If all they want is a script that updates the Version string, we can do that, and that's less likely to break.  I feel like often I just manually change the file when someone complains, but it looks like I edited the cronjob in april, so I'm not sure.

**Comment by azurebrd** [comment]
@valearna fixed.  somehow a variable was the wrong variable, not sure how that typo happened, that script shouldn't have been able to generate anything while being tested, with that typo.

**Comment by azurebrd** [comment]
@raymond91125 @valearna   For the original ticket, I've updated the file mentioned on the ticket (tea.cgi on wobr) to use the correct URL, but I don't have permissions for git on there.  I don't remember if I should or I shouldn't.  Also I'm on wobr2 and this says to do it on wobr1, and I don't recall how the proxies and live/dev server stuff works with wobr.  Either way though, if someone else has access to the git repo and can change things, it's a literal change on line 389 to the URL on this ticket.
